client/client.go

```diff
@@ -383,9 +383,15 @@ func WithExecPath(execPath string) OpOption {
 	}
 }
 
-func WithWhitelistedSubnets(whitelistedSubnets string) OpOption {
+func WithWhitelistedSubnets(trackSubnets string) OpOption {
```

Comment:
Why are we keeping this one?

Reply:
For back compatibility. That is part of the ANR client library API (eg used by CIs). I think we can keep it and plan to remove it in a couple of releases in the future.